recipes/com/graphql-java/scm.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 type: "git"
-uri: "https://github.com/graphql-java/graphql-java-extended-scalars"
+uri: "https://github.com/graphql-java/graphql-java.git"
+tagMapping:
```

Comment:
Is this mapping required? The system should be smart enough to figure out simple mappings like this on its own.

Reply:
The logic is here: https://github.com/redhat-appstudio/jvm-build-service/blob/main/java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/location/LookupScmLocationCommand.java#L128

Basically because this tag exactly contains the version it should be used with no user input required. The only way this will fail is if there are similar tags that confuse it (e.g. v1.0 and b1.0).